apps/explorer/src/ui/Amount.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+export type AmountProps = { amount: number; coinSymbol?: string };
+
+export function Amount({ amount, coinSymbol }: AmountProps) {
```

Comment:
`Amount` might be a little too generic. I called this `CoinBalance` in the Jira (this partially solves this Jira task: https://mysten.atlassian.net/browse/APPS-123 although that task is for a slightly different UI so we'll still need to follow-up and make this take a variant). Alternative name could be something like `FormattedCoin`.